src/components/input/input.html

```diff
@@ -35,7 +35,7 @@
 
       <label class="md-input-placeholder"
              [attr.for]="inputId"
-             [class.md-empty]="empty"
+             [class.md-empty]="empty && type !== 'date'"
```

Comment:
Alright, I decided to do that in the template since I saw you guys did that with some other things. But it's fixed now in the latest commit.